pages/learn/develop/container-contracts.md

```diff
@@ -76,13 +76,13 @@ Could not move to new release: Some releases were rejected due to having unmet r
 Contracts: Services with unmet requirements: first-service
 ```
 
-**Note** For application fleets that contain a mixture of Supervisor and L4T versions, contracts may be used to ensure only compatible devices run specific services, as releases will only be deployed to those meeting the contract requirements.
```

Comment:
This is a good one, what on earth is an 'application fleet' 😆

Reply:
I sure don't wanna know them.